src/test/java/com/exasol/adapter/dialects/oracle/OracleScalarFunctionsIT.java

```diff
@@ -0,0 +1,157 @@
+package com.exasol.adapter.dialects.oracle;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.*;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import com.exasol.adapter.commontests.scalarfunction.ScalarFunctionsTestBase;
+import com.exasol.adapter.commontests.scalarfunction.TestSetup;
+import com.exasol.adapter.commontests.scalarfunction.virtualschematestsetup.*;
+import com.exasol.adapter.commontests.scalarfunction.virtualschematestsetup.request.Column;
+import com.exasol.adapter.commontests.scalarfunction.virtualschematestsetup.request.TableRequest;
+import com.exasol.adapter.metadata.DataType;
+import com.exasol.closeafterall.CloseAfterAll;
+import com.exasol.closeafterall.CloseAfterAllExtension;
+import com.exasol.dbbuilder.dialects.Schema;
+import com.exasol.dbbuilder.dialects.Table;
+import com.exasol.dbbuilder.dialects.exasol.VirtualSchema;
+import com.exasol.dbbuilder.dialects.oracle.OracleObjectFactory;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static com.exasol.adapter.metadata.DataType.IntervalType.DAY_TO_SECOND;
+import static com.exasol.matcher.ResultSetStructureMatcher.table;
+
+@ExtendWith({CloseAfterAllExtension.class})
+class OracleScalarFunctionsIT extends ScalarFunctionsTestBase {
+    @CloseAfterAll
+    private static final OracleVirtualSchemaIntegrationTestSetup SETUP = new OracleVirtualSchemaIntegrationTestSetup();
+    static int idCounter = 0;
+
+    protected static String getUniqueIdentifier() {
+        ++idCounter;
+        return "ID" + idCounter;
+    }
+
+    @Override
+    protected TestSetup getTestSetup() {
+        final OracleObjectFactory oracleFactory = SETUP.getOracleFactory();
+
+        return new TestSetup() {
+
+            @Override
+            public VirtualSchemaTestSetupProvider getVirtualSchemaTestSetupProvider() {
+                return (final CreateVirtualSchemaTestSetupRequest request) -> {
+                    //create schema in oracle DB with increasing ID, ID1, ID2, etc.
+                    final Schema oracleSchema = oracleFactory.createSchema(getUniqueIdentifier());
+
+                    for (final TableRequest tableRequest : request.getTableRequests()) {
+                        // TODO; check this (done, was not fine)
```

Comment:
what does this mean?

Reply:
that it wasn't ok as I first thought but I deleted the todo 😅